webconsole/statik.go

```diff
@@ -10,7 +10,7 @@ import (
1+\xb4\xc5\x0cf\x98\x866\x8dK\x80\x81Sc\x88\xb1\xe0P;R!\x9c\x8b\xc4\xbe\x83\n\x14+\xf8\xbcw\x82\x8a\xbf\xd5(S\xecF\x14\x89\xa8\xe5\x9c\xd1;\x80R\xf3\"R\xd0\xa6*\xd3)[\xd0\xacx\xba\xf9\x84n|+\xde\x04\x16U5hD-	=\x8e+\x18\xffd]\xd3\xd3%,\xe1H-\xa2^\xe0\x02\xcf\x12\x90\x1cW\xcc\x95\xfb\x8eE<\xe3\xf6\x9c]+R$[1\xd99\x8d\x8d\xd5\x18,N\x94\x92)\x13\x07l\xe6\x9bC}\xff\x99\x87\x00\x98\xe5\xba\x14'\xe4\x08\xf2W\x8b\xb7f]\x18^\xe1 \xc6O\xf3Jg\xe75#\xf0G\x80L\x00\x8c%\x0d\x1b\xd2\xa8\x9e\xe7\xc6\xb2G5\xa5\xc62\x81\x90\x9b\xdd\x89\x8b^\x9c\xcaF\xb5\x0f\xe2\x80\x90\xfdD\x0dw\x19\xaa\x89\xe2\x1cA\x07\xc1#Q\xa6\xd9\x8d\x08\xba\xd4)\x8f\xb8bq{AE\x9c\xdf\x06\x97\x11\xf8\xc7N\xb5	\xc0\xa1xh\x81Dp\xa2\xbf\x98qV\xc8\x0fC\xfb\x8d,\xfc$nK\x93\x8b\xea\xdbw\x81\xc6x\xefO4L\xcd.\xe2\x10\xe5s*1\xc5\xa3\xf3\xf7\xa3`oa+\xb5i\x05B\x04\x00\x1b\x8e\xc8\x1f\x11\x05\x02\xa1\xa5\xbew\xd4\xd8Mj\xd0\x8b\xef\xaf\x81\x90\xcc\xc3q\xc5\xcb8A\x03\xc3XS6\xb0\xb66\x14\x8a\xd1\xd6\xff\xf5&\xc9)l2\x01\xba\x90\xee/\xe7\xe2\xeb\x96\x89\x18\xa9\xe2	9\x0d\x1d\xd2\xaa\xf8%\xfa9\x13\x82\xdd\x89)d\xb0\xcax]\x9f\x07<\xc89_%y\xf6,\x9a\x9b\xd5\xf8\xbd;\xaaE;\x84\x1d\xff\xbc\xd1<\x92\xdd\x8a\x99\xb3\x06 d\x99\x02\x18\x18\xedW\x13\xc9\x02\x1e\xb2\xb2\x0e\x17\xa3\x9b\\?\xb9YF\x95\xf5\xc5\xeca3\x01\xf6h\xf9\xc9\x19\xa0\xa3Ck\x90\xa2vJ\x95\xe9\x02:\x0b/\xc8}W:\x0f.>/\xfa?i\xc2\xc1\xa9\x82\x0b_\xf9\x94\xd25M\x9cQ\xa8\xaf\xa8'\x169\x0f+\x10]W\xce\xa0\xed\x18\xb9jEx\xde\xd8\xdb\xde\x9b9\xbb\xa8u\xe5\x1c'/5\xd0O\xfd5J\x8f\x8fx\x07\xb6O\xca\x93fy\xb8\xb6\xac\x0e\xd3\x85\x90\xaf\x123\x82uw\xa4If\x0d\xcb*\xe05u\xc1\xe8\xda\xe0bI\xb4\x07>Y\xe7\xadZ\xac\x88)m,5A	\xd4\x87\xfc\xfd5Dr\xcff\xcdaQ\xc9a\x8b\x11\xca\x89W\x1d\x9b\xd4I\xd8\x9b\x81\x07\xef%`\x10\x00\x12\x88\xf6\xc0P\xd8\xa4Q\xb8t\xca\xd8\xd9\xa1\x14\xa4\x0e\xff_\xa7\xfd'St_U-\xb0m^\xfc\xe0\xff\xdbG\xfe\xf0/t\xe7\xc6\xed\x1fZ\xcc\xce!\x941\xe3\xf0\xddX\xe7\xab\x8e\xf0\xd0\xee\x80\xe56\x16^\xb3 \x8f3\xf8\xe9Q,\xa2\x04\x07\xf3\xd6\xcc\xad\x8b\xef\xeb\x14\x0b\x1e\xbe\xe5\xa7\x07\xa9\xcf\xa3\x8e\xd3r\x8f\xdf\\\x96O7\x0b\xa3\xed\x85\xd0\xf1\x01c\xfd\xd1.\xce\xfa\xf0WO\xc1\xbb\x98\xc77\xd7\x1d\x95T\x81\xaf\xae\x1f\x1ek\xd4\xac\xb7\x99\xee\xd1\xc1\x17`\x87S\xb8Zq\n+\xf9y\xb2o\x12\xf1\xdc\x99&/O&;\xdf\xd4\x99&^\x1c\xb9qsH(\xc3\xbb\xa6\x86\xd4\xb2\x80{\xdc\x08\xfd2\x11b\xb2\x1f+\xa6\x9c\xf4\xaf\x8f\xe9\xa4\xa4N\xdf]\xcb\x9f\xca\x05\x17\xffy\xe4>~\xebI\x00\xfa\x9d\xca\xca\x1elr\x1df\xb7-!\xf9\xd2q\xc7\x12\x0b\x0b\x0bKCU\x07Q\xafl\x13\xf1?\x01\x00\x00\xff\xffPK\x07\x08{\x86\x02V\x89p\x00\x00\x99p\x00\x00PK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\xe4HES\xaf\x13j\x89\x12\x02\x00\x00\x14\x04\x00\x00\x12\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xb4\x81\x00\x00\x00\x00missing/index.htmlUT\x05\x00\x01\xbd\x15\\aPK\x01\x02\x14\x03\x14\x00\x08\x00\x08\x00\xabe\xdeR{\x86\x02V\x89p\x00\x00\x99p\x00\x00\x12\x00	\x00\x00\x00\x00\x00\x00\x00\x00\x00\xb4\x81[\x02\x00\x00missing/mascot.pngUT\x05\x00\x01bg\xdc`PK\x05\x06\x00\x00\x00\x00\x02\x00\x02\x00\x92\x00\x00\x00-s\x00\x00\x00\x00"
```

Comment:
Hello @SimoneLazzaris !

This statik file is not committed usually. Possible to exclude it?

Reply:
Excluded in commit 2071adbe